services/discordService.js

```diff
@@ -0,0 +1,30 @@
+const jwt = require("jsonwebtoken");
+
+const DISCORD_BASE_URL = config.get("services.discordBot.baseUrl");
+
+const getDiscordMembers = async () => {
+  const authToken = jwt.sign({}, config.get("rdsServerlessBot.rdsServerLessPrivateKey"), {
+    algorithm: "RS256",
+    expiresIn: config.get("rdsServerlessBot.ttl"),
+  });
+
+  try {
+    const response = await fetch(`${DISCORD_BASE_URL}/discord-members`, {
+      method: "GET",
+      headers: { "Content-Type": "application/json", Authorization: `Bearer ${authToken}` },
+    })
+      .then((response, err) => {
+        return response.json();
+      })
```

Comment:
Nit : we can use async/await instead of promise chaining

Reply:
tried, but wasn't working